src/common/config.cc

```diff
@@ -635,6 +640,11 @@ Config::Config(std::string jsonfilename)
   // Agora configurations
   frames_to_test_ = tdd_conf.value("max_frame", 9600);
   core_offset_ = tdd_conf.value("core_offset", 0);
+  if (dynamic_core_allocation_) {
+    worker_thread_num_ = sysconf(_SC_NPROCESSORS_ONLN) - (core_offset_ + socket_thread_num_ + dynamic_core_allocation_ + 1); // use all available cores
+  } else {
+    worker_thread_num_ = tdd_conf.value("worker_thread_num", 25);
+  }
   worker_thread_num_ = tdd_conf.value("worker_thread_num", 25);
```

Comment:
this won't work, probably need to remove line 648.  Name 25 while you are editing